exercises/nodejs/src/game.js

```diff
@@ -1,172 +1,208 @@
-import generator from 'random-seed'
+import generator from "random-seed";
 
 var Game = function () {
-  var players = new Array()
-  var places = new Array(6)
-  var purses = new Array(6)
-  var inPenaltyBox = new Array(6)
+  var players = new Array();
+  var boardSquares = new Array(6);
+  var playerCoins = new Array(6);
+  var inPenaltyBox = new Array(6);
 
-  var popQuestions = new Array()
-  var scienceQuestions = new Array()
-  var sportsQuestions = new Array()
-  var rockQuestions = new Array()
+  var popQuestions = new Array();
+  var scienceQuestions = new Array();
+  var sportsQuestions = new Array();
+  var rockQuestions = new Array();
 
-  var currentPlayer = 0
-  var isGettingOutOfPenaltyBox = false
+  var currentPlayer = 0;
+  var isGettingOutOfPenaltyBox = false;
 
   var didPlayerWin = function () {
-    return !(purses[currentPlayer] == 6)
-  }
+    return !(playerCoins[parseInt(currentPlayer)] === 6);
+  };
 
   var currentCategory = function () {
-    if (places[currentPlayer] == 0) { return 'Pop' }
-    if (places[currentPlayer] == 4) { return 'Pop' }
-    if (places[currentPlayer] == 8) { return 'Pop' }
-    if (places[currentPlayer] == 1) { return 'Science' }
-    if (places[currentPlayer] == 5) { return 'Science' }
-    if (places[currentPlayer] == 9) { return 'Science' }
-    if (places[currentPlayer] == 2) { return 'Sports' }
-    if (places[currentPlayer] == 6) { return 'Sports' }
-    if (places[currentPlayer] == 10) { return 'Sports' }
-    return 'Rock'
-  }
+    const popCategory = "Pop";
+    const scienceCategory = "Science";
+    const sportsCategory = "Sports";
+
+    const editors = {
```

Comment:
This is certainly clearer than the if statements. I would make it more intentful by declaring that Rock is in position 3, 7 and 11 too